feat(global): :sparkles: fix minHeight & margin in AccordionSummary &…

PritamBag opened the pull request:
## Description
- Removed 48px minHeight from AccordionSummary.
- Eliminated default 12px margin in.MuiAccordionSummary-content.
- Ensured toolbox takes full height.

## Related Issues
Ref #167

## Testing
<!-- Describe the testing steps you have taken to ensure that your changes work as expected -->

## Checklist

- [x] I have performed a thorough self-review of my code.
- [ ] I have added or updated relevant tests for my changes.
- [ ] My code follows the project's style guidelines and best practices.
- [ ] I have updated the documentation if necessary.
- [ ] I have added or updated relevant comments in my code.
- [ ] I have resolved any merge conflicts of my branch.


## Screenshots (if applicable)
<!-- Include screenshots or animated GIFs to visually demonstrate the changes, if applicable -->

## Additional Notes

<!--Feel free to add any other relevant information that might be helpful to reviewers.-->

## Reviewers

<!--Tag any specific individuals or teams you'd like to review this pull request.

Thank you for your contribution!-->

---

## Maintainer Notes

- [ ] Has this change been tested in a staging environment?
- [ ] Does this change introduce any breaking changes or deprecations?

PritamBag committed:
feat(global): :sparkles: fix minHeight & margin in AccordionSummary & full height in toolbox

Removed 48px minHeight from AccordionSummary, Eliminated default 12px margin in .MuiAccordionSummary-content, Ensured toolbox takes full height

Ref #167

anantakumarghosh commented on package/nativeComponents/surfaces/NativeToolBox.js:
fix component file name

anantakumarghosh replied:
resolved

anantakumarghosh commented on package/styledComponents/surfaces/SCToolBox.js:
fix component file name

anantakumarghosh replied:
resolved

anantakumarghosh commented on package/index.js:
```diff
@@ -154,7 +154,7 @@ import NativeFooter from "./nativeComponents/surfaces/NativeFooter";
 import NativeRightDrawer from "./nativeComponents/surfaces/NativeRightDrawer";
 import NativeToolbar from "./nativeComponents/surfaces/NativeToolbar";
-import NativeToolBox from "./nativeComponents/surfaces/NativeToolBox.js";
+import NativeToolbox from "./nativeComponents/surfaces/NativeToolbox";
```
check filename

anantakumarghosh replied:
resolved

anantakumarghosh committed:
fix(global): :rotating_light: fix component file name

fix component file name, changed NativeToolBox to NativeToolbox and SCToolBox to SCToolbox

ref #167
